src/Microsoft.AspNetCore.Mvc.Razor/Compilation/CompiledViewManifest.cs

```diff
@@ -12,13 +11,7 @@ public static class CompiledViewManfiest
     {
         public static readonly string PrecompiledViewsAssemblySuffix = ".PrecompiledViews";
 
-        public static Type GetManifestType(AssemblyPart assemblyPart, string typeName)
-        {
-            var assembly = GetFeatureAssembly(assemblyPart);
-            return assembly?.GetType(typeName);
-        }
-
-        private static Assembly GetFeatureAssembly(AssemblyPart assemblyPart)
+        public static Assembly GetFeatureAssembly(AssemblyPart assemblyPart)
```

Comment:
We have tests via `ViewsFeatureProvider`: https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.Razor.Test/Compilation/ViewsFeatureProviderTest.cs#L94